.github/workflows/site.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy Site to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+        - 'docs/**'
+  pull_request:
```

Comment:
Will this publish the docs on PR's?

Reply:
No -- the deploy step is skipped unless the trigger is a push to main.

Reply:
Oh, I see it now